BedrockServer.cpp

```diff
@@ -176,16 +179,27 @@ void BedrockServer::sync(const SData& args,
 
     // Initialize the DB.
     int64_t mmapSizeGB = args.isSet("-mmapSizeGB") ? stoll(args["-mmapSizeGB"]) : 0;
-    SQLite db(args["-db"], args.calc("-cacheSize"), true, args.calc("-maxJournalSize"), -1, workerThreads - 1, args["-synchronous"], mmapSizeGB, args.test("-pageLogging"));
 
-    // And the command processor.
+    // We allocate 25% of the total available file handles to our DB pool. Note that on Linux, the default limit is only
+    // 2014 file handles, so this can be as low as 256 if not adjusted elsewhere (it's adjusted in main.cpp for live
```

Comment:
2014?  I thought it was 1024?

Reply:
Yes, it's 1024, this is an extreme character transposition. I'll fix the comment.